src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChartControls.tsx

```diff
@@ -65,13 +90,15 @@ const ReleaseChartControls = ({summary, yAxis, onYAxisChange}: Props) => {
         <SectionValue key="total-value">{summary}</SectionValue>
       </InlineContainer>
 
-      <OptionSelector
-        title={t('Y-Axis')}
-        selected={yAxis}
-        options={yAxisOptions}
-        onChange={onYAxisChange as (value: string) => void}
-        menuWidth="150px"
-      />
+      {yAxisOptions.length > 1 && (
+        <OptionSelector
+          title={t('Y-Axis')}
+          selected={yAxis}
+          options={yAxisOptions}
+          onChange={onYAxisChange as (value: string) => void}
```

Comment:
Why do you have to cast this?

Reply:
onYAxisChange takes parameter of type enum YAxis but that shared component OptionSelector accepts only string